checker/jtreg/nullness/type-argument/TypeArgument.out

```diff
@@ -0,0 +1,4 @@
+TypeArgument.java:10:12: compiler.err.proc.messager: [method.invocation.invalid] call to foo() not allowed on the given receiver.
```

Comment:
Also the `type-argument` directory seems like overkill for these two files. Just put it together with the other tests one level higher.

Reply:
Okay, I will delete the folder.